src/xenia/emulator.cc

```diff
@@ -314,32 +315,35 @@ X_STATUS Emulator::TerminateTitle() {
   return X_STATUS_SUCCESS;
 }
 
-std::string Emulator::CanonicalizeFileExtension(
-    const std::filesystem::path& path) {
-  return xe::utf8::lower_ascii(xe::path_to_utf8(path.extension()));
-}
-
-const std::unique_ptr<vfs::Device> Emulator::CreateVfsDeviceBasedOnPath(
+const std::unique_ptr<vfs::Device> Emulator::CreateVfsDeviceBasedOnSignature(
```

Comment:
If you really want to call it like that then you have to explicitly provide signature as parameter.
Right now it is misleading as name assumes that it will be added on usage